README.md

```diff
@@ -1,237 +1,127 @@
 <a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
--->
-
 <div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] Your README logo link is broken. Kindly fix that. If it is the Microverse logo kindly remove it entirely or replace it with yours.

  ![image](https://user-images.githubusercontent.com/36542986/228175898-3689fa53-baaa-4f71-91ac-a21ea8d75a94.png)